pinot-core/src/main/java/org/apache/pinot/core/data/table/IndexedTable.java

```diff
@@ -33,43 +33,41 @@
  */
 public abstract class IndexedTable implements Table {
 
-  List<AggregationFunction> _aggregationFunctions;
+  AggregationFunction[] _aggregationFunctions;
   int _numAggregations;
   DataSchema _dataSchema;
 
-  int _maxCapacity;
+  int _capacity;
   int _bufferedCapacity;
 
+  boolean _isOrderBy;
+  IndexedTableResizer _indexedTableResizer;
+
   @Override
   public void init(@Nonnull DataSchema dataSchema, List<AggregationInfo> aggregationInfos, List<SelectionSort> orderBy,
```

Comment:
No specific reason. I think since we designed the interface first, we ended up starting with an init.
I  changed it to constructor (read up about it and realized, as better practice, should not be using init if it can be done in constructor)